specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -657,7 +657,7 @@ def recover_original_data(eval_shifted_extended_evaluation: Sequence[BLSFieldEle
 
     reconstructed_data = bit_reversal_permutation(fft_field(reconstructed_poly, roots_of_unity_extended))
 
-    return reconstructed_data
+    return CosetEvals(reconstructed_data)
```

Comment:
CosetEvals is just 64 field elements. `reconstructed_data` here is likely the full extended length 8192.

Reply:
Yeah you are right -- this was diffed over from 3697 so it should be an issue there too